components/GlobalStyle/sub-components/Typography.jsx

```diff
@@ -55,21 +54,13 @@ ul, ol {
   padding-left: 18px;
 }`;
 
-const NunitoFontFamily = createGlobalStyle`
-  @import url('https://fonts.googleapis.com/css?family=Nunito+Sans:400,600,700,400i,600i,700i&display=swap');
-`;
-
 const Typography = () => (
   <>
     <Style />
-    {isSSR() ? (
```

Comment:
**question:** is there motivation to remove this set to SSR? Now, will CSR and SSR use it?

Reply:
Yes, the styled-components > 5 does not support font imports via css, only via html.
https://styled-components.com/releases#v5.0.0